backend/qemu.pm

```diff
@@ -402,8 +404,13 @@ sub save_snapshot {
     });
 
     mkpath(VM_SNAPSHOTS_DIR);
+    my $migrate_compress_level = 9;
+    # ARM is slow, and high compression level is much slower cpu wise than
+    # disk speed, so we tradeoff faster migration over higher compression
+    $migrate_compress_level = 2 if if ($vars->{ARCH} eq 'aarch64' || $vars->{ARCH} eq 'arm');
```

Comment:
I wonder if `$vars->{QEMU_SNAPSHOT_COMPRESSION} // 2` would be better and have that setting defined in the workers.ini, as this might affect other architectures/machines where more workers are running and there's not so much space (and document it on backend_vars.asciidoc file in the meantime)

Reply:
I think 9 should be the default because x86_64 and Power8 have no issue with it, but yeah, it should be configurable.

Also there appears to be two ifs.

Reply:
sure they don't have "an issue" but both x86 and power 8 have significantly faster I/o than this compression is ever gonna save. 

note that this is compressing 4k pages at a time, not a stream compression. while I haven't been able to do elaborate comparisons, a stream gzip on compression level of 1 (the least best but most fast compression) is ten fold (!) better compressing than qemu's builtin compression at level 9. I would argue that it isn't worth the cpu overhead on any architecture, but since that is probably contentious by the openqa maintainers I limited it to aarch64.

Reply:
@dirkmueller I get the thing of x86 not having an issue, but the actual change makes sense to me, just with $vars->{QEMU_SNAPSHOT_COMPRESSION} // 2 :)